### ISSUE TITLE
Does this not work with Rust stable?

### ISSUE THREAD

**Issue by moranbw** [twitchax/rtz#8]
I get the following when trying to use the library:

`#![feature] may not be used on the stable release channel`

For several lines in lib.rs: 
```
#![feature(async_closure)]
#![feature(test)]
#![feature(string_remove_matches)]
#![feature(fs_try_exists)]
#![feature(once_cell)]
```

**Comment by moranbw** [comment]
Updated my rust to 1.80.  No longer getting this error in rust-analyzer.  But now getting the following when I have the lib added in my `Cargo.toml` like this:

```
rtz = { version = "0.5.3", features = [ "full", "self-contained" ] }
```

![image](https://github.com/user-attachments/assets/c34c61db-f046-44da-acc2-6ba06a8fe39d)

If I can provide any more context let me know.  Just trying to import the library at this point.

**Comment by twitchax** [comment]
Interesting on the second bit.  As for the stable bit, I think I could fix it to work with stable.  Do you require stable at this point?

**Comment by moranbw** [comment]
Honestly not sure what is going on.  When I have `Cargo.toml` like this:

```
rtz = "*" #choose a version
```
I get the stable release error.  When it is like this:
```
rtz = { version = "0.5.3", features = ["full"] } 
```
I get the unresolved / gated error.

For our application we would definitely be looking to stay on Rust stable.

Your library seems really promising...we operate in an offline environment, and it would be very beneficial to capture local time of our users.  We have access to GPS time+coordinates, so this library could allow us to solve a whole slew of issues.

**Comment by moranbw** [comment]
I have a minimal example working of my own without all of the bells and whistles here.  So if this is not a priority for you, no problem.  But if you want to keep this issue open as a marker for getting to Stable, that works too.  But I am unblocked.

Thanks for creating this library regardless.  It has been interesting to peruse and always happy to see the Geo ecosystem in Rust expand.

**Comment by twitchax** [comment]
Awesome: great to hear.  Happy to hop on a call for any use case help, as well.

Let's leave this open since I think I can get it to stable pretty easily.  Started out as a pet project, so leaned into features, but adapted into this once I realized I had something by pretty cool going.

**Comment by twitchax** [comment]
Latest release is now pinned to MSRV `1.80`.
